chrome_manifest_v3/content-script.js

```
@@ -21,48 +21,47 @@ var globalInit = false;
-            memory: [],
-            count: 0,
-            max: 100,
-            debugMode: false,
-        };
-        chrome.storage.sync.get("debugMode", function (response) {
-            if (response.hasOwnProperty("debugMode") && response["debugMode"]) {
-                window.g2t_log_g.debugMode = true;
-            }
-        });
-    }
+  if (!window.hasOwnProperty('g2t_log_g')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)